### PR TITLE
Add Translation QUEST_LV_300 Lines 11-39; 61-99

### PR DESCRIPTION
**Pull request by HelpfulKoreanGirl**

### CODE REVIEW

**Comment by stvkno** [comment]
i think QUEST_LV_0300_20150717_000012/14/16/17 will go off the screen due to length? @sunaries

**Comment by Sunaries** [comment]
@stvkno Probably, we should just leave it like that. 
Though, make sure to add a space before the parenthesis. E.g. .... (1)

**Comment by HelpfulKoreanGirl** [comment]
@Sunaries I double checked, but there's no space before the parentheses in the Korean text. Should I still add a space?

**Comment by HelpfulKoreanGirl** [comment]
Is there a current Issue where we should be posting what lines we're working on?

**Comment by Sunaries** [comment]
@HelpfulKoreanGirl There is indeed no space between the parentheses, but in English we should add a space.

**Comment by HelpfulKoreanGirl** [comment]
Added spaces before parentheses ^^

Also added lines 61-84

**Comment by stvkno** [comment]
> Švitrigaila (Polish: Świdrygiełło; before 1370 – 10 February 1452) was the Grand Duke of Lithuania from 1430 to 1432

man, i knew i wasn't crazy for not being able to transliterate that from korean to english lol

**Comment by HelpfulKoreanGirl** [comment]
Added translations for lines 85-100

@stvkno LOL I FEEL YOUR PAIN

File: QUEST_LV_0300.txt [hunk 1]
```diff
@@ -58,45 +58,45 @@ QUEST_LV_0300_20150717_000057
...
+QUEST_LV_0300_20150717_000091 You gave Svitrigaila the poisonous herbs, and it looked like he ate them, but you don't see any change in him. Try speaking to him again.
+QUEST_LV_0300_20150717_000092 Seeing as there was no effect on Svitrigaila when he ate the poisonous herbs, it must be that he needs to mix them with holy water first, and then eat them to get the desired result. Find the Cleric Master and get the holy water from him.
+QUEST_LV_0300_20150717_000093 Destroy the Terminal
+QUEST_LV_0300_20150717_000094 Talk with Svitrigaila some more.
+QUEST_LV_0300_20150717_000095 Destroy the Soul Compactor Terminal
+QUEST_LV_0300_20150717_000096 In order to help Svitrigaila, you'll have to destroy the Soul Compaction Device. You'll need to gather some tools to help you destroy the Soul Compaction Device terminal, and then its the mainframe, beforehand. First things first, destroy the first terminal.
+QUEST_LV_0300_20150717_000097 Inform Svitrigaila
+QUEST_LV_0300_20150717_000098 You have destroyed the terminal. And you successfully retrieved the terminal's core. Give the core to Svitrigaila.
+QUEST_LV_0300_20150717_000099 Destroy the Terminal and Retrive its Core
```
Comment (Ragunaga) [inline QUEST_LV_0300.txt:99]: Retrive should be Retrieve

Just a small little typo I noticed

File: QUEST_LV_0300.txt [hunk 2]
```diff
@@ -1,42 +1,42 @@
...
-QUEST_LV_0300_20150717_000031
-QUEST_LV_0300_20150717_000032
-QUEST_LV_0300_20150717_000033
-QUEST_LV_0300_20150717_000034
-QUEST_LV_0300_20150717_000035
-QUEST_LV_0300_20150717_000036
-QUEST_LV_0300_20150717_000037
-QUEST_LV_0300_20150717_000038
-QUEST_LV_0300_20150717_000039
+QUEST_LV_0300_20150323_000001	$To support the Knights of Kaliss, we were deployed here from headquarters.{nl} If you want to know more, head over to our headquarters at Verkti Square.
```
Comment (Ragunaga) [inline QUEST_LV_0300.txt:1]: To make the first sentence flow better, you may want to switch around the two clauses to something like;
"Headquarters deployed us here to support the Knights of Kaliss."

File: QUEST_LV_0300.txt [hunk 3]
```diff
@@ -1,42 +1,42 @@
...
+QUEST_LV_0300_20150323_000002	$I've been a merchant all my life. When Medzio Diena hit, I thought of setting up shop here as I heard it was safest here.
+QUEST_LV_0300_20150323_000003	$I met many wandering Revelators around here. I really think they were...different, and it wasn't just that they were strong.
```
Comment (Ragunaga) [inline QUEST_LV_0300.txt:3]: I believe 'think' should be changed to 'thought' in this context. May be wrong, does the original korean have it in its present tense?

File: QUEST_LV_0300.txt [hunk 4]
```diff
@@ -1,42 +1,42 @@
...
+QUEST_LV_0300_20150323_000004	$I don't mean to burden the Goddess' priests even more. I just think blindly believing and waiting is a little too much for us normal folk.
+QUEST_LV_0300_20150323_000005	$The seals of this village are closed right now, so something something might've tampered with it? I think the priest might just be paranoid about it all.
```
Comment (Ragunaga) [inline QUEST_LV_0300.txt:5]: You doubled the word something here.

I think might have should be changed to may have. Both are right, but generally might have is used for situations where something could have possibly happened, but didn't.

"The enemy might have ambushed us in that alleyway, if we hadn't gotten out so quick."

File: QUEST_LV_0300.txt [hunk 5]
```diff
@@ -1,42 +1,42 @@
...
+QUEST_LV_0300_20150323_000006	$I hope the priests don't prioritize the seal above everything else here. It sometimes seems like they'd even sacrifice themselves for it.
+QUEST_LV_0300_20150323_000007	$The villagers could feel all sorts of things if the priest was more transparent about everything.{nl} Maybe that's why as much as we have to endure...
```
Comment (Ragunaga) [inline QUEST_LV_0300.txt:7]: I'm not entirely sure what the sentence is trying to say.

File: QUEST_LV_0300.txt [hunk 6]
```diff
@@ -1,42 +1,42 @@
...
+QUEST_LV_0300_20150323_000008	$The evil energy is making this forest decay into a lifeless place.{nl}It is easily the biggest problem this village faces as of now.
+QUEST_LV_0300_20150323_000009	$Goddess...{nl}Please tell me the seal of the village is still okay...
+QUEST_LV_0300_20150323_000010	$From now there will surely still be demons trying to attack and bring down the seal.{nl}I wonder how long will it be before Goddess Austeja can return...
```
Comment (Ragunaga) [inline QUEST_LV_0300.txt:10]: Generally in this sort of grammar structure, "it will be" would be preferred instead of "will it be."

Comment (HelpfulKoreanGirl) [inline QUEST_LV_0300.txt:10]: Sorry @Ragunaga i didn't translate these lines >.< @vtange translated 1-10

Comment (HelpfulKoreanGirl) [inline QUEST_LV_0300.txt:10]: I guess I can edit them on my PR#1344 if you like. I edited Line #1 based on your suggestion in my new PR, but these lines have already been merged by imcGames so i thought they're okay.

Comment (Ragunaga) [inline QUEST_LV_0300.txt:10]: Oh they merged it already, didn't think they had? Ohwell then. You can just ignore them.

**Pull request closed without merging**